react-app/src/components/OutputDisplay.js

```diff
@@ -7,7 +7,7 @@ function OutputDisplay(props) {
   return (
     <div>
       {props.id === null && !props.showLogs && !props.showSaveForm && (
-        <p className="page-text padding-100">
+        <p className="page-text text-centered padding-100">
           Select a patient to view their information or click on "Log File" to view the logger messages.
```

Comment:
Super small, but could we change this to say 'click on "Logger Messages" to view...'